scanner/image/scanner/download-mappings.sh

```diff
@@ -2,8 +2,23 @@
 
 set -euo pipefail
 
-output_dir="/mappings"
-mkdir $output_dir
+if [[ "$#" -lt "1" ]]; then
+  >&2 echo "Error: please pass target directory as a command line argument."
+  exit 1
+fi
```

Comment:
Please ensure compatibility with the other usage of this script in https://github.com/stackrox/stackrox/blob/master/scanner/image/scanner/Dockerfile#L11 .

Reply:
Done :+1: